helm/aws-ebs-csi-driver-app/templates/snapshot-controller-pdb.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.enableVolumeSnapshot }}
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: ebs-snapshot-controller
+  namespace: kube-system
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+spec:
+  minAvailable: 1
+  selector:
+    matchLabels:
+      app: ebs-snapshot-controller
+      {{- include "labels.common" . | nindent 6 }}
+      {{- if .Values.snapshotController.podLabels }}
```

Comment:
do we need all these labels on the pdb ? 

personally i prefer to keep the minimum set of labels for selectors to avoid that `versions` label break the pdb when there is an upgrade

Reply:
you convinced me despite your questionable football team choice

Reply:
:wolf: > :imp: